modules/geolocation/outputanalysis.py

```diff
@@ -0,0 +1,40 @@
+import numpy as np
+
+def get_best_location(array):
+
+    tempArrayOne=array[:,[2]].copy()
+    tempArrayTwo=array[:,[0]].copy()
+    tempArrayThree=array[:,[1]].copy()
+
+#splits the coordinate pairs, error/radius and confidence from the input numpy array into three arrays
```

Comment:
Comments should be at the same indent level as the code it is clarifying. Also, make sure to add a space after the # symbol e.g. `# My comment`.

Reply:
Also comments should be immediately above the code it clarifies, not below.